lib/revision-selector/index.jsx

```diff
@@ -125,6 +125,10 @@ export class RevisionSelector extends Component {
       'is-visible': isViewingRevisions,
     });
 
+    if (!isViewingRevisions) {
```

Comment:
I suppose we can't move this higher in the function because linting? It would be nice to short-circuit some of the above logic sooner.

Reply:
Done, not a linting issue.